task5/src/main/java/com/clouway/adapter/persistence/PersistentFundsRepository.java

```diff
@@ -89,4 +92,64 @@ public boolean withdraw(Double amount, String email) {
     }
     return succesfull;
   }
+
+  public void updateHistory(String date, String email, String operation, Double amount) {
```

Comment:
You are right. I made date save as Long - System.currentTimeInMillis(), so it will not depend on the time zone.